src/main/java/com/alibaba/ttl/TransmittableThreadLocal.java

```diff
@@ -14,6 +17,10 @@
  * <p>
  * Note: {@link TransmittableThreadLocal} extends {@link java.lang.InheritableThreadLocal},
  * so {@link TransmittableThreadLocal} first is a {@link java.lang.InheritableThreadLocal}.
+ * <p>
+ * If you have netty in the runtime and {@link io.netty.util.internal.FastThreadLocal} is supported. You can store
```

Comment:
`TTL` is **standalone** and **small** lib.

So `TTL` can NOT be dependent to `netty` lib(it's **SOoOoO** big!), even at **runtime**.

\# It's ok to port(copy) the netty code, as long as the code is concise.